addons/cookoff/stringtable.xml

```diff
@@ -7,15 +7,15 @@
             <Czech>Povolit explozi munice</Czech>
             <Russian>Включить воспламенение</Russian>
             <Japanese>誘爆を有効化</Japanese>
-            <Polish>Aktywuj efekty zapłonu</Polish>
+            <Polish>Aktywuj efekty wybuchu amunicji</Polish>
         </Key>
         <Key ID="STR_ACE_CookOff_enable_tooltip">
             <English>Enables cook off and related vehicle destruction effects.</English>
             <German>Ermöglicht Selbstzündung und zugehörige Fahrzeug-Zerstörungseffekte.</German>
             <Czech>Povolí explozi munice a její následné ničivé efekty.</Czech>
             <Russian>Включает воспламенение и сопутствующие эффекты повреждения техники.</Russian>
             <Japanese>誘爆を有効化し、車両が誘爆によって破壊されていきます。</Japanese>
-            <Polish>Aktywuje efekt zapłonu na zniszczonych pojazdach.</Polish>
+            <Polish>Aktywuje efekt wybuchu amunicji na zniszczonych pojazdach.</Polish>
```

Comment:
BTW , i changed some previously translated things, like "Aktywuje efekt zapłonu "
for " Aktywuje efekt wybuchu amunicji" because i think that this is less accurate transaltion of the single word (coof- off is in polish zagotowanie-zapłon) but "wybuch amunicji" makes more sense as we take the military-specific language :)